pkgs/servers/dns/knot-resolver/manager.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, knot-resolver, writeText, python3 }:
+
+with python3.pkgs;
+
```

Comment:
```suggestion
```

Reply:
Moved `with` to the `*Inputs` definitions.  (and ran nixfmt)